app/controllers/users_controller.rb

```diff
@@ -7,6 +7,9 @@ def show
 
   def update
     current_user.telephone = phone_params
```

Comment:
Should phone_params be strong params, using `require` and `permit`? Or is that effectively what phone_params are, just a different way of writing those?